app/assets/stylesheets/generic/timeline.scss

```diff
@@ -0,0 +1,150 @@
+.timeline {
+  list-style: none;
+  padding: 20px 0 20px;
+  position: relative;
+}
+
+.timeline:before {
+  top: 0;
+  bottom: 0;
+  position: absolute;
+  content: " ";
+  width: 3px;
+  background-color: #eeeeee;
```

Comment:
Color literals like `#eeeeee` should only be used in variable declarations; they should be referred to via variable everywhere else.